### DISCUSSION TITLE
Change the controls in the editor

### DISCUSSION THREAD

**Discussion by RajathShanmugaiah** [react-page/react-page#1087] [General]
![Capturenew](https://user-images.githubusercontent.com/63197365/138450196-cf8b9c8e-52af-4fb1-b7ea-389c0d0a54e9.PNG)
Is there a way I can pick only few controls out of these as I don't require resize and move option in my project.

**Comment by macrozone** [comment]
@RajathShanmugaiah currently not! but should be easy to add, i opened a issue for that: https://github.com/react-page/react-page/issues/1088

**Reply by RajathShanmugaiah** [reply]
Okay thank you Ill access it once it is updated 👍

**Reply by RajathShanmugaiah** [reply]
I found a short term fix as of now by doing 

  
```
useEffect(() => {
    if(process.browser) {
      let userControl: NodeListOf<Element> = document.querySelectorAll(".react-page-controls-mode-toggle-control");
    userControl = document.querySelectorAll(".react-page-controls-mode-toggle-control");
    let moveControl = userControl[3];
    let resizeControl = userControl[4];
    moveControl?.setAttribute("style","display:none");
    resizeControl?.setAttribute("style", 'display:none');
    }
  },[])

```

I hope others can find this solution as a short term fix
